syntax = "proto3";

enum OrderStatus{
    PENDING = 0;
    SHIPPED = 1;
    DELIVERED = 2;
}

message Order{
    int32 id  = 1;
    int64 created_at = 2; 
    int64 updated_at = 3;
    string username = 4;
    string email = 5;
    string product_name = 6;
    int32 quantity = 7;
    float price = 8;
    OrderStatus status = 9;
}




